posthog/hogql_queries/query_runner.py

```diff
@@ -74,48 +74,48 @@
 EXTENDED_CACHE_AGE = timedelta(days=1)
 
 
-class ExecutionMode(IntEnum):  # Keep integer values the same for Celery's sake
```

Comment:
I was concerned about this line, but potential renaming seems to be not an issue anymore.

Reply:
Indeed, we aren't actually using these values over Celery, and if we will in the future, strings are naturally going to be stable